tests/smoketest/go.mod

```diff
@@ -2,6 +2,16 @@ module github.com/fluxcd/golang-with-libgit2/tests/sample
 
 go 1.18
 
+// A temporary fork of git2go was created to enable use
+// of libgit2 without thread support to fix:
+// fluxcd/image-automation-controller/#339.
+//
+// This can be removed once libgit2/git2go#918 is merged.
```

Comment:
Please link the current PR to the git2go one. We may want to move this fork under fluxcd org if it's not going to be accepted upstream.